core/lib/config/src/configs/da_dispatcher.rs

```diff
@@ -14,6 +14,9 @@ pub struct DADispatcherConfig {
     pub max_rows_to_dispatch: Option<u32>,
     /// The maximum number of retries for the dispatch of a blob.
     pub max_retries: Option<u16>,
+    /// Enable on-chain verification of the data availability.
+    #[serde(default)]
+    pub enable_onchain_verification: bool,
```

Comment:
`serde(default)` doesn't work for file-based configs, because they use custom protobuf-based library for deserilization instead of `serde` so this is a breaking change. IIRC there is a way to make default work for it too, but I don't remember it.
A safe way would be to wrap the field in `Option` and provide a getter.

Reply:
Also, is `serde(default)` safe here? It would make the value `false` by default, meaning that all the clients will omit inclusion info. I would suggest using the safest value as a default.

Reply:
done, switched to getter with `true` as a default value